hack/check-everything.sh

```diff
@@ -50,6 +50,14 @@ SKIP_FETCH_TOOLS=${SKIP_FETCH_TOOLS:-""}
 function fetch_kb_tools {
   local dest_dir="${1}"
 
+  # use the pre-existing version in the temporary folder if it matches our k8s version
+  if [[ -x "${dest_dir}/kubebuilder/bin/kube-apiserver" ]]; then
+    version=$("${dest_dir}"/kubebuilder/bin/kube-apiserver --version)
+    if [[ $version == *"${k8s_version}"* ]]; then
+      return 0
```

Comment:
should we write something to stdout in this case? That we're using already existing version